data/virtualization/autoyast/host_12.xml.ep

```diff
@@ -0,0 +1,323 @@
+<?xml version="1.0"?>
+<!DOCTYPE profile>
+<profile xmlns="http://www.suse.com/1.0/yast2ns" xmlns:config="http://www.suse.com/1.0/configns">
+  <bootloader>
+    <global>
+      <activate>true</activate>
+      <gfxmode>auto</gfxmode>
+      <hiddenmenu>false</hiddenmenu>
+      <os_prober>false</os_prober>
+      <serial>serial --unit=1 --speed=115200 --parity=no</serial>
+      <terminal>serial</terminal>
+      <timeout config:type="integer">10</timeout>
+      <trusted_grub>false</trusted_grub>
+      <xen_append>splash=silent quiet console=ttyS1,115200</xen_append>
```

Comment:
Some worker needs ttyS0 while some needs ttyS1, right?

Reply:
The workers just use the value of SERIALDEV.  
Xen Kernel will dump more information to serial port during boot if console=ttySx is appended here. We can add a console=ttyS0,115200 or remove "console=ttyS1" here.

Reply:
It seems to bring some misunderstanding. Let me explain again. If I understand correctly, the xen_append in autoyast will affect the grub.cfg that is generated after host installation. However, the console setting about ttySx should be directly related to which physical com port the serial console is binded to. For all MU machines, most machines uses com0 => ttyS0, while the new amd machines use com1 => ttyS1. So to let all machines console setting to be correct, the xen_append here should be dynamically set rather than hard code to ttyS0 or ttyS1, right?  You can also refer to http://openqa.suse.de/tests/8162489/modules/logs_from_installation_system/steps/1/src line 59 for complete xen serial console setup.

Reply:
Check if the new change is "dynamic" :)